settings.gradle

```diff
@@ -3,12 +3,12 @@ pluginManagement {
 		resolutionStrategy {
 			eachPlugin {
 				if (requested.id.id == 'bio.terra.test-runner-plugin') {
-					useModule("bio.terra:terra-test-runner:" + requested.version)
+					useModule("bio.terra:terra-test-runner:${requested.version}")
 				}
 			}
 		}
```

Comment:
This can all go away if they resolve https://broadinstitute.slack.com/archives/C020RSQF5KL/p1630534822020700